src/base_model.jl

```diff
@@ -103,26 +113,26 @@ function build_base_model!(ECModel::AbstractEC, optimizer; use_notations=false)
     # Design of assets of the user
     @variable(model_user,
         0 <= x_us[u=user_set, a=device_names(users_data[u])]
-            <= field_component(users_data[u], a, "max_capacity"))
+            <= field_component(users_data[u], a, "max_capacity")/field_component(users_data[u], a, "nom_capacity"), Int)
 
     ## Expressions
 
     # CAPEX by user and asset
     @expression(model_user, CAPEX_us[u in user_set, a in device_names(users_data[u])],
-        x_us[u,a]*field_component(users_data[u], a, "CAPEX_lin")  # Capacity of the asset times specific investment costs
+        x_us[u,a]*field_component(users_data[u], a, "CAPEX_lin")*field_component(users_data[u], a, "nom_capacity")  # Capacity of the asset times specific investment costs
     )
 
     @expression(model_user, CAPEX_tot_us[u in user_set],
         sum(CAPEX_us[u, a] for a in device_names(users_data[u])) # sum of CAPEX by asset for the same user
     )  # CAPEX by user
 
-    @expression(model_user, C_OEM_us[u in user_set, a in device_names(users_data[u])],
-        x_us[u,a]*field_component(users_data[u], a, "OEM_lin")  # Capacity of the asset times specific operating costs
-    )  # Maintenance cost by asset
+    @expression(model_user, C_OEM_us[u in user_set, a in asset_names_ex(users_data[u],[THER,LOAD])],

```

Comment:
Mmm rather than creating a special class, we could keep the same formulation also for the thermal and having 2 terms of OEM one linear (default value 0.0) and one by commitment.